.github/workflows/master_check_label.yml

```diff
@@ -0,0 +1,50 @@
+#After getting teams name required from codeowners check labels if present for that team, if not create a label for that team
+---
+name: Check Changed Files
+
+on:
+  workflow_call:
+    inputs:
+      required-teams:
+        description: 'Teams required for approval in the PR'
+        required: true
+        type: string
+
+  workflow_dispatch:
+
+jobs:
+  check-labels:
+    runs-on: self-hosted
+    steps:
+      - name: Check all labels
```

Comment:
You are assuming that it will be teams only ; But if the `CODEOWNERS` file has username this will fail

Reply:
We already discussed this in api-spec repo that having specific users won't be a good decision in case the user leaves, so better to have a separate team for that user